mne/stats/cluster_level.py

```diff
@@ -944,20 +944,24 @@ def permutation_cluster_test(X, threshold=None, n_permutations=1024,
         2 groups with respectively 20 and 17 observations in each.
         Each data point is of shape (50, 4).
     threshold : float | dict | None
-        If threshold is None, it will choose a t-threshold equivalent to
-        p < 0.05 for the given number of (within-subject) observations.
+        If threshold is None, it will choose a f-threshold equivalent to
```

Comment:
"an F threshold" - also, I suggest to use F instead of f everywhere in the docs.